scripts/pkg_in_pipe.py

```diff
@@ -54,18 +60,29 @@ def print_table_footer(out):
 </div>
 ''', file=out)
 
-def print_table_line(out, build, link, by):
+def print_table_line(out, build, link, issues, by):
+    issues_content = '\n'.join([f'<li><a href="https://project.vates.tech/vates-global/browse/XCPNG-{i['sequence_id']}/">XCPNG-{i['sequence_id']}</a></li>' for i in issues])
     print(f'''    
             <tr class="odd:bg-white odd:dark:bg-gray-900 even:bg-gray-50 even:dark:bg-gray-800 border-b dark:border-gray-700 border-gray-200">
                 <th scope="row" class="px-6 py-4 font-medium text-gray-900 whitespace-nowrap dark:text-white">
                     <a href="{link}">{build}</a>
                 </th>
+                <td class="px-6 py-4">
+                    <ul>
+                    {issues_content}
+                    </ul>
+                </td>
                 <td class="px-6 py-4">
                     {by}
                 </td>
             </tr>
 ''', file=out)
 
+# load the issues from plane, so we can search for the plane card related to a build
+resp = requests.get('https://project.vates.tech/api/v1/workspaces/vates-global/projects/43438eec-1335-4fc2-8804-5a4c32f4932d/issues/',
+                     headers={'x-api-key': os.environ['PLANE_TOKEN']})
```

Comment:
Such requirements on envvars, as well as new package requirements, should be documented in a nearby README.
That could be a reason to use a separate `scripts/` subdir, so a specific README  can be kept with it?